packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -277,7 +265,8 @@ export function TokenApprovalDialog(props: TokenApprovalDialogProps) {
             <InformationCircleIcon className="h-6 w-6 text-cyan-dark" />
             <span className="text-sm font-light text-cyan-dark">
               After approval, you&apos;ll see a second prompt in your wallet for{' '}
-              {noteMessage}{' '}
+              the {isCctp ? 'CCTP' : 'standard'} {destinationNetworkName}{' '}
+              {isDepositMode ? 'deposit' : 'withdrawal'} fee.
               <ExternalLink
```

Comment:
that's a lot of copy lol, and tbh it sounds kinda crapy, for example:

```
After approval, you'll see a second prompt in your wallet for the CCTP Arbitrum One deposit fee.
```

what if we just simplified to 

```
After approval, you'll see a second prompt in your wallet for the deposit/withdrawal transaction.
```